toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -1325,9 +1325,30 @@ func configureUserStartupCommand(installChroot *safechroot.Chroot, user configur
 }
 
 func provisionUserSSHCerts(installChroot *safechroot.Chroot, user configuration.User, homeDir string) (err error) {
+	var (
+		pubKeyData            []string
+		exists                bool
+	)
 	const squashErrors = false
+	const authorizedKeysTempFilePerms = 0644
+	const authorizedKeysTempFile = "/tmp/authorized_keys"
 
 	userSSHKeyDir := filepath.Join(homeDir, ".ssh")
+	authorizedKeysFile := filepath.Join(homeDir, ".ssh/authorized_keys")
+
+	exists, err = file.PathExists(authorizedKeysTempFile)
+	if err != nil {
+		logger.Log.Warnf("Error accessing %s file : %v", authorizedKeysTempFile, err)
+		return
+	}
+	if !exists {
+		logger.Log.Warnf("File %s does not exist. Creating file...", authorizedKeysTempFile)
```

Comment:
This sounds like an expected scenario. If that's the case, then don't log in on the warning level and use debug instead. Otherwise we end up displaying noise to the user and it's making debugging of real issues harder.

Reply:
Done